packages/docs/components/sidebar/Sidebar.vue

```diff
@@ -1,64 +1,162 @@
 <template>
-  <va-sidebar :minimized="minimized">
-    <template slot="menu">
-      <template v-for="(item, key) in items">
-        <va-sidebar-link-group
-          v-if="item.children"
+  <va-sidebar class="sidebar" color="secondary" v-model="minimized">
```

Comment:
Sidebar links are not loaded in nuxt.
![image](https://user-images.githubusercontent.com/5394573/95446070-7a6acb00-0968-11eb-820a-b508aab00503.png)